Package.swift

```diff
@@ -14,15 +14,16 @@ let package = Package(
     ],
     dependencies: [
         .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.9.0-beta.1")),
-        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("23.1.0-beta.1")),
+        // We keep MME dependency for compatibility reasons
```

Comment:
Create a ticket to remove it in the next major version?

Reply:
Done: MAPSIOS-383